tests/test_parser.py

```diff
@@ -257,6 +259,67 @@ def test_convert_to_date_cast_result_to_date(self, mocked_locale,
                          expected_output)
 
 
+class TestParseParentPage(unittest.TestCase):
+    @patch('smarsy.parse.BeautifulSoup')
+    def test_parent_page_content_called_with_expected_html(self,
+                                                           mocked_soup):
+        html = '<tr></tr>'
+        parent_page_content_to_object(html)
+        mocked_soup.assert_called_with(html, 'html.parser')
+
+    @patch('bs4.BeautifulSoup', side_effect=ValueError)
+    def test_parent_page_content_raise_exeption_with_wrong_file_format(
```

Comment:
Also the test name doesn't correspond to the actual action. There is no file, there is html document.

Reply:
this is not done